src/esp/physics/bullet/BulletRigidObject.h

```diff
@@ -431,6 +431,8 @@ class BulletRigidObject : public BulletBase,
    * updates. See @ref btRigidBody::setWorldTransform. */
   void syncPose() override;
 
+  std::string getBulletDebugName();
```

Comment:
Put in BulletBase.h as pure virtual and override here please.

Reply:
This version of getBulletDebugName (with no arguments) is specific to BulletRigidObject. It is not needed by the other classes that inherit from BulletBase.